src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Pages/manage-services/Service-Detail.cshtml.cs

```diff
@@ -24,6 +24,8 @@ public class Service_DetailModel : ServicePageModel
     private readonly IServiceDirectoryClient _serviceDirectoryClient;
     private readonly ITaxonomyService _taxonomyService;
 
+    public bool UserRoleCanDeleteService => $"{RoleGroups.LaManagerOrDualRole},{RoleGroups.VcsManagerOrDualRole}".Contains(FamilyHubsUser.Role);
```

Comment:
This might get interned by the framework, but perhaps we could put:

$"{RoleGroups.LaManagerOrDualRole},{RoleGroups.VcsManagerOrDualRole}"

into a const or static variable.

Might make readability/maintainability easier

Reply:
Sounds good, resolved in: [155e9e7](https://github.com/DFE-Digital/fh-services/pull/148/commits/155e9e757df605c95e19fcbd3a44934093cb1571)